Sources/Cocoa/Helpers/Constants.swift

```diff
@@ -118,9 +118,17 @@ public enum AppConstants {
     public static var φ: CGFloat { 0.618 }
 
     public static var statusBarHeight: CGFloat {
-        UIApplication
-            .sharedOrNil?
-            .firstSceneKeyWindow?
+        // On some situations we need the bar height even when no scene is active
+        // (e.g when showing biometric screen). In those situations we resort
+        // to not filtering for an active scene.
+        guard let firstSceneWindow =
+                UIApplication.sharedOrNil?.firstSceneKeyWindow() ??
+                UIApplication.sharedOrNil?.firstSceneKeyWindow(activeScene: false)
+        else {
+            return 44 // TODO: 44 is outdated. have some logic to determine device and correct height?
+        }
+
+        return firstSceneWindow
             .windowScene?
             .statusBarManager?
             .statusBarFrame.height ?? 44
```

Comment:
This should work without any of those changes too, right? 🤔 

.statusBarFrame.height ?? (isPhoneNotch ? 44 : 20)